### PR TITLE
Upgrade graphics packages to LF6.1.55_2.2.0

### PR DESCRIPTION
**Pull request by valijec**
This PR upgrades the following recipes:
1. `imx-gpu-g2d`: Updated from version `6.4.11.p2.0` to `6.4.11.p2.2`
2. `kernel-module-imx-gpu-viv`: Updated from version `6.4.11.p2.0` to `6.4.11.p2.2`
3. `kernel-module-isp-vvcam`: Updated from `4.2.2.22.0` to `4.2.2.24.0`
4. `imx-pxp-g2d`: Upgraded to revision `f3adab05b`
5. `imx-gpu-viv`: Updated from `6.4.11.p2.0` to `6.4.11.p2.2`
6. `weston`: Updated from `11.0.2` to `11.0.3`

### CODE REVIEW

File: recipes-graphics/wayland/weston_11.0.3.imx.bb [hunk 1]
```diff
@@ -162,8 +162,8 @@ DEFAULT_PREFERENCE = "-1"
 SRC_URI:remove = "https://gitlab.freedesktop.org/wayland/weston/uploads/f5648c818fba5432edc3ea63c4db4813/${BPN}-${PV}.tar.xz"
 SRC_URI:prepend = "git://github.com/nxp-imx/weston-imx.git;protocol=https;branch=${SRCBRANCH} "
 SRC_URI += "file://0001-Revert-protocol-no-found-wayland-scanner-with-Yocto-.patch"
-SRCBRANCH = "weston-imx-11.0.2"
-SRCREV = "a3c5e63aa94f8d27b72780e333370be78c9a7653"
+SRCBRANCH = "weston-imx-11.0.3"
+SRCREV = "a509c319b7bfc72638c9b9d8f5e4b6259aba0ab7"
```
Comment (thochstein) [inline recipes-graphics/wayland/weston_11.0.3.imx.bb:166]: You also need to update `PREFERRED_VERSION_weston` in `imx-base.inc`.

Comment (valijec) [reply recipes-graphics/wayland/weston_11.0.3.imx.bb:166]: Thank you @thochstein , I missed that :)

**Comment by quaresmajose** [comment]
look like you have added merge commit on the PR on the last push, can you please rebase our branch on master to fix it?

**Review by otavio — CHANGES_REQUESTED** [review]
Please add changelog and commits to commit logs.

**Comment by valijec** [comment]
> @valijec news
Sorry, missed the comment, i'll add them in a few minutes, Thank you :)

**Pull request merged**